Views/frontend/plugins/nosto_tagging/index.tpl

```diff
@@ -60,7 +60,7 @@
           //<![CDATA[
           {literal}
           if (typeof Nosto === 'undefined') {
-            const Nosto = {};
+            var Nosto = {};
```

Comment:
Can you add a comment on the code? We'll forget and remove when refactoring this